### PR TITLE
Add kazzna profile for 106th

### PR DESCRIPTION
**Pull request by kazzna**
Update profiles for meetup tonight.

### CODE REVIEW

File: members/kazzna.md [hunk 1]
```diff
@@ -23,19 +23,33 @@ Webアプリを作りたくて勉強し始めたら、書籍がRubyかPHPばか
...
 ### Rails 4
-* プログラミング全般では「ほんば」以上
-
-システム会社で社内の教育係などもやっていました。
-
-* ネット関係(Web等)は「たね」
+* 「ふたば」
 
-ネットの繋がらない職場でしかプログラミングをしたことが有りません。
+HTMLを少し書けるようになってきました！！
```
Comment (yucao24hours) [inline members/kazzna.md]: 💐

File: members/kazzna.md [hunk 2]
```diff
@@ -23,19 +23,33 @@ Webアプリを作りたくて勉強し始めたら、書籍がRubyかPHPばか
...
 
 ### GitHub
-* 「ふたば」
+* 「ほんば」
 
-ひとり用です。
+昨年末についにPull Requestを送信しました！！
```
Comment (yucao24hours) [inline members/kazzna.md]: 🎉

**Comment by yucao24hours** [comment]
LGTM :rose:

**Comment by kadoppe** [comment]
LGTM!

**Pull request merged**